plasma_framework/contracts/mocks/exits/payment/outputGuardHandlers/PaymentOutputGuardHandlerWrapper.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "../../../../src/exits/models/OutputGuardModel.sol";
+import "../../../../src/exits/payment/outputGuardHandlers/PaymentOutputGuardHandler.sol";
+
+/**
+For unkown reason truffle is not able to pass in the struct of OutputGuardModel.Data as input args directly.
```

Comment:
Try passing a tuple instead

Reply:
after testing, tuple failed but array works. 
I also tried, if we return a struct from contract, it would be represented as an `Array`.